packages/shared/components/inputs/IconTextInput.svelte

```diff
@@ -0,0 +1,15 @@
+<script type="typescript">
+    import { Icon, TextInput, InputContainer } from 'shared/components'
+
+    export let value: string
+    export let icon: string
+    export let inputElement: HTMLInputElement
+    export let isFocused = true
+</script>
+
+<InputContainer bind:this={inputElement} bind:inputElement col classes="space-y-2">
+    <div class="flex flex-row w-full items-center space-x-0.5">
```

Comment:
Do we need this div or can we pass in row instead on col to InputContainer and add the needed classes?